src/grouped-legend-styles.ts

```diff
@@ -1,11 +1,11 @@
-import { Chart, ChartOptions } from 'chart.js';
-import { valueOrDefault } from 'chart.js/helpers';
+import { Chart, ChartOptions, FontSpec } from 'chart.js';
+import { resolve, toFont } from 'chart.js/helpers';
 
 export class Styles {
-  private static chartDefaults: ChartOptions;
+  private static chartOptions: ChartOptions;
 
-  static setDefaultsFromChart(chart: Chart): void {
-    this.chartDefaults = chart.options;
+  static initialize(chart: Chart): void {
```

Comment:
What if instead of `initialize()` and accessing the static getters, you build an object with these styling fields? I'm still not a big fan of this static stuff

Reply:
As we discussed on Slack, it's not that simple, but I've changed it to normal class, so styles are not shared between charts, as it had some potential to cause bugs in the future.

Reply:
Alright, that makes more sense.